postgres-batch.yaml

```
@@ -0,0 +1,50 @@
+id: postgres-batch
+namespace: company.team
+
+tasks:
+  - id: download_products_csv_file
+    type: io.kestra.plugin.core.http.Download
+    uri: https://huggingface.co/datasets/kestra/datasets/raw/main/csv/products.csv
+
+  - id: products_csv_to_ion
+    type: io.kestra.plugin.serdes.csv.CsvToIon
+    from: "{{ outputs.download_products_csv_file.uri }}"
+
+  - id: postgres_create_table
+    type: io.kestra.plugin.jdbc.postgresql.Query
+    url: "jdbc:postgresql://{{ secret('POSTGRES_HOST') }}:25432/postgres"
```

Can we use the default postgres port? This won't clash with the Kestra DB as that doesn't expose the port by default

```suggestion
    url: "jdbc:postgresql://{{ secret('POSTGRES_HOST') }}:5432/postgres"
```